Tests/Functional/Drivers/Queue/StompQueueDriverTest.php

```diff
@@ -76,4 +76,123 @@ function fwrite()
+     * @param MessageInterface $msg
+     */
+    public function testAfterTtlShouldDiscard(MessageInterface $msg)
+    {
+        $queueMessage = $this->createQueueMessage($msg);
+        $queueMessage->setTTL(1);
+        $this->driver->subscribe($this->queueName);
+        $this->driver->send($queueMessage);
+        $this->driver->unSubscribe();
+
+        // After > 2 seconds, the message is not there
+        \sleep(2);
+        $this->driver->subscribe($this->queueName);
+        $msgOut = $this->driver->receive();
+        if ($msgOut) {
+            $this->driver->ack();
+        }
+        $this->assertNull($msgOut);
+
+        $this->driver->disconnect();
+    }
+
+    public function testItShouldGetManyMessagesAtOnceUsingPrefetchSize()
+    {
+        // populate the queue with some messages
+        $numMessages = 3;
+        $sentMessages = [];
+        for ($i = 0; $i < $numMessages; ++$i) {
+            $message = $this->createQueueMessage($this->createSimpleEntity('item'.$i));
+            $this->driver->send($message);
+            $sentMessages[] = $message;
+        }
+
+        $this->driver->subscribe($this->queueName, null, $numMessages);
+
+        $receivedMessages = [];
+        while (null !== ($receivedMessage = $this->driver->receive())) {
+            $this->driver->ack();
+            $receivedMessages[] = $receivedMessage;
+        }
+
+        $this->assertSame(
+            \array_map('self::mapTitle', $sentMessages),
+            \array_map('self::mapTitle', $receivedMessages)
+        );
+    }
+
+    public function testPrefetchSizeShouldWorkAlsoWhenNotReceivingAllThePrefetchedMessagesAtOnce()
+    {
+        // populate the queue
+        $prefetchSize = 3;
+        $numMessagesSent = 10;
+        $sentMessages = [];
+        for ($i = 0; $i < $numMessagesSent; ++$i) {
+            $message = $this->createQueueMessage($this->createSimpleEntity('item'.$i));
+            $this->driver->send($message);
+            $sentMessages[] = $message;
+        }
+
+        $this->driver->subscribe($this->queueName, null, $prefetchSize);
+
+        $receivedMessages = [];
+
+        // receive just one message
+        $receivedMessages[] = $this->driver->receive();
+        $this->driver->ack();
+
+        // send an additional message to the queue
+        $message = $this->createQueueMessage($this->createSimpleEntity('item'.$numMessagesSent));
+        $this->driver->send($message);
+        $sentMessages[] = $message;
+
+        // receives all the remaining messages in the queue
+        while (null !== ($receivedMessage = $this->driver->receive())) {
+            $this->driver->ack();
+            $receivedMessages[] = $receivedMessage;
+        }
+
+        $this->assertSame(
+            \array_map('self::mapTitle', $sentMessages),
+            \array_map('self::mapTitle', $receivedMessages)
+        );
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function createConsumer(): ConsumerInterface
```

Comment:
This method is not used in the class, it can be deleted

Reply:
Done.